wandb/sdk/artifacts/storage_handlers/local_file_handler.py

```diff
@@ -110,8 +110,6 @@ def md5(path: str) -> B64MD5:
                     # `LogicalPath` returns a "logical path"; it's a relative path
                     # **on the local filesystem**.
                     logical_path = os.path.relpath(physical_path, start=local_path)
-                    if name is not None:
-                        logical_path = os.path.join(name, logical_path)
 
                     entry = ArtifactManifestEntry(
                         path=logical_path,
```

Comment:
I think you might need to update this to include the `name` parameter now that we're removing it from `logical path`..?  Otherwise `name` won't be respected even in the artifact paths. I think we only want to change how the reference uri is formed, not the manifest path, right?

Reply:
ah, good point.

Reply:
thanks again for catching that!